charts/sm-operator/values.yaml

```diff
@@ -0,0 +1,62 @@
+settings:
+  # How often the secrets synchronize in seconds.  Minimum value is 180.
+  bwSecretsManagerRefreshInterval: 300
+  # Cloud region for sync.  Please see: https://bitwarden.com/help/families-for-enterprise-self-hosted/#step-1-enable-cloud-communication
+  cloudRegion: US
```

Comment:
Just curious because I am not sure how this works - what if they want to use EU? Is that configurable?

Reply:
Yes.  That is one of the options in the schema.